docs/extensibility/implement-security-between-integrations.md

```diff
@@ -1,39 +1,36 @@
 ---
-title: Implement auth from custom resource to integration
-description: Learn how to implement authentication credentials from a custom resource to a custom integration.
-ms.date: 08/12/2024
+title: Implement security between hosting and client integrations
+description: Learn how to implement security between hosting and client integrations.
+ms.date: 09/12/2024
 ms.topic: how-to
 ---
 
-# Implement auth from custom resource to integration
+# Implement security between hosting and client integrations
```

Comment:
"Securely connect hosting and client integrations" maybe? "Implement security" feels weird but also I'm definitely being dramatic

Reply:
We landed on "Secure communication between hosting and client integrations".